constrain/library/vav_turndown_during_reheat.py

```diff
@@ -49,8 +49,8 @@ def verify(self):
             self.df["V_dot_VAV_max"] > 0
         ).all(), "Not all `V_dot_VAV_max` values are greater than 0"
 
-        # Check if the `reheat_coil_flag` column has only False values
-        if (self.df["reheat_coil_flag"] == False).all():
+        # Check if the `reheat_coil_flag` column has only True/False values
+        if self.df["reheat_coil_flag"].nunique() == 1:
```

Comment:
If only reheat mode or not reheat mode data are available, this verification item can't be tested. Thus, these cases were updated to ```Untested```.